src/main/java/bio/terra/cli/command/user/sshkey/Generate.java

```diff
@@ -22,8 +23,15 @@ public class Generate extends BaseCommand {
       description = "Save the terra ssh key pair as file, skip printing out the key")
   boolean saveToFile;
 
+  @CommandLine.Mixin ConfirmationPrompt confirmationPromptOption;
+
   @Override
   protected void execute() {
+    confirmationPromptOption.confirmOrThrow(
+        "Generating a new Terra Ssh key will replace the old Terra SSH key if it exists. "
+            + "You must disassociate the old SSH public key from your GitHub account using https://github.com/settings/keys. "
```

Comment:
I feel like this is misleading because you don't really have to do this. Maybe the message should be more proactive, like "You must add the new public key to your GitHub account..."